lightning/src/chain/keysinterface.rs

```diff
@@ -80,7 +80,7 @@ pub enum SpendableOutputDescriptor {
 	/// To derive the remote_revocation_pubkey provided here (which is used in the witness
 	/// script generation), you must pass the remote revocation_basepoint (which appears in the
 	/// call to ChannelKeys::on_accept) and the provided per_commitment point
-	/// to chan_utils::derive_public_revocation_key.
+	/// to chan_utils::derive_public_broadcaster_revocation_key.
```

Comment:
The comments/variables/etc here still have a ton of references to remote_, to_self_delay, etc.

Reply:
Right corrected, note there is still remote/local references around SpendableOutputDescriptors as they're relative to ChannelMonitor (and I'll do post-anchor to avoid a confusing rebase).